velox/docs/monthly-updates/august-2022.rst

```diff
@@ -0,0 +1,80 @@
+******************
+August 2022 Update
+******************
+
+Documentation
+=============
+
+* Add documentation for :func:`combinations` Presto function.
+* Add documentation for :doc:`/functions/window`.
+* Add documentation for :func:`array_sort`.
+* Add `website`_ for Velox.
+
+.. _website: https://velox-lib.io
+
+Core Library
+============
+
+* Add support for Right Semi Join.
```

Comment:
Looks like we need to update https://facebookincubator.github.io/velox/develop/joins.html

Reply:
Yes, will do it in a follow-up PR.